src/devices/machine/spi_sdcard.h

```diff
@@ -12,16 +12,16 @@ class spi_sdcard_device : public device_t
 public:
 	// SPI 4-wire interface
 	auto spi_miso_callback() { return write_miso.bind(); }
-	void spi_clock_w(int state);
-	void spi_ss_w(int state) { m_ss = state; }
-	void spi_mosi_w(int state) { m_in_bit = state; }
+	void spi_clock_w(bool state);
+	void spi_ss_w(bool state) { m_ss = state; }
+	void spi_mosi_w(bool state) { m_in_bit = state; }
```

Comment:
The standard signature for “line” handlers in MAME uses `int` to represent the line state.  You can cast it to `bool`, but please don’t use non-standard signatures for handlers.